{"data_mtime": 1588885990, "dep_lines": [1], "dep_prios": [5], "dependencies": ["builtins"], "hash": "6c575be54daba1cf6d08dc94e3f46bad13579b4092c9bce674e8bc93ab6b74c4", "id": "services.user_management.exceptions", "ignore_all": false, "interface_hash": "297c3953445d81460e8636424880cf3ef1077aa9543e773669872e205a39e368", "mtime": 1588558488, "options": {"allow_redefinition": false, "allow_untyped_globals": false, "always_false": [], "always_true": [], "bazel": false, "check_untyped_defs": false, "disallow_any_decorated": false, "disallow_any_explicit": false, "disallow_any_expr": false, "disallow_any_generics": false, "disallow_any_unimported": false, "disallow_incomplete_defs": false, "disallow_subclassing_any": false, "disallow_untyped_calls": false, "disallow_untyped_decorators": false, "disallow_untyped_defs": false, "follow_imports": "normal", "follow_imports_for_stubs": false, "ignore_errors": false, "ignore_missing_imports": false, "implicit_reexport": true, "local_partial_types": false, "mypyc": false, "no_implicit_optional": false, "platform": "linux", "plugins": [], "show_none_errors": true, "strict_equality": false, "strict_optional": true, "strict_optional_whitelist": null, "warn_no_return": true, "warn_return_any": false, "warn_unreachable": false, "warn_unused_ignores": false}, "path": "services/user_management/exceptions.py", "plugin_data": null, "size": 159, "suppressed": [], "version_id": "0.770"}